.vscode/launch.json

```diff
@@ -5,24 +5,25 @@
   "version": "0.2.0",
   "configurations": [
     {
-      "name": "flutter_template_riverpod",
+      "name": "flutter_template_riverpod(development mode)",
       "request": "launch",
       "type": "dart",
-      "program": "lib/main/main_development.dart"
-    },
-    {
-      "name": "flutter_template_riverpod (profile mode)",
-      "request": "launch",
-      "type": "dart",
-      "flutterMode": "profile",
-      "program": "lib/main/main_production.dart"
+      "program": "lib/main/main_development.dart",
+      "args": [
+        "--flavor",
+        "Development",
+       ]
     },
     {
       "name": "flutter_template_riverpod (release mode)",
       "request": "launch",
       "type": "dart",
       "flutterMode": "release",
-      "program": "lib/main/main_production.dart"
+      "program": "lib/main/main_production.dart",
+      "args": [
+        "--flavor",
+        "Release",
```

Comment:
The 2 Flavors are "Development" and "Production".
You are confusing between "flavors" and "build modes" (`flutterMode`) by calling this "release".
So please rename this flavor to "Production".

Please refer to the docs : https://docs.flutter.dev/testing/build-modes

So in total there will be 6 app versions...
Development-Debug
Development-Profile
Development-Release
Production-Debug
Production-Profile
Production-Release

However I acknowledge that this `launch.json` currently only has 2 app versions listed.
I think it would be nice to add a "Development-Debug" launch option for VSCode users actually, because this is most common.

Reply:
@diarmaidlindsay i am keeping only two not including profile just to make same as android.

Reply:
build mode i kept only two debug and release
you want me to include profile mode? i ignored profile to keep same as android
https://docs.flutter.dev/testing/build-modes

Reply:
>So please rename this flavor to "Production".

changed.